payments-core/src/main/java/com/stripe/android/paymentsheet/ui/AddPaymentMethodsFragmentFactory.kt

```diff
@@ -15,8 +15,8 @@ internal class AddPaymentMethodsFragmentFactory<ViewModelType : BaseSheetViewMod
 ) : FragmentFactory() {
     override fun instantiate(classLoader: ClassLoader, className: String): Fragment {
         return when (className) {
-            AddCardFragment::class.java.name -> {
-                AddCardFragment(viewModelClass, viewModelFactory)
+            CardDataCollectionFragment::class.java.name -> {
```

Comment:
We will have to look at it when we talk, but I think you will just be able to pass in a Form, there won't be anything payment method specific.